portfolio/src/main/webapp/images.html

```diff
@@ -9,6 +9,10 @@
     <body>
         <div id="content">
             <h1>Sibing's Image Gallery</h1>
+            <p>To be honest I did not understand what "paragraphs explaining what you enjoy photographing" means.
+               I guess it means "in the pictures you showed above, what did you enjoy".
+            </p>
+            <script src="paragraph.js"></script>
             <div id="gallery">
```

Comment:
Do we still need the image href below if we can construct everything dynamically from the script?

Reply:
It is the design. I design in a way that user can access the picture from both the "P1" heading and the image itself.